bitnami/spring-cloud-dataflow/Chart.yaml

```diff
@@ -38,4 +38,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-spring-cloud-dataflow
   - https://github.com/bitnami/bitnami-docker-spring-cloud-skipper
   - https://dataflow.spring.io/
-version: 2.3.0
+version: 2.3.1
```

Comment:
Could we increase this to 2.4.0? Since it includes important changes to the actual configuration of the app.